test/features/pagecontent/pagecontent.js

```diff
@@ -310,40 +310,6 @@ describe('item requests', function() {
     //        assert.deepEqual(res.status, 200);
     //    });
     //});
-
-    it('should redirect to commons for missing file pages', function() {
-        return preq.get({
-            uri: server.config.bucketURL + '/html/File:ThinkingMan_Rodin.jpg'
-        })
-        .then(function(res) {
-            assert.deepEqual(res.status, 200);
-            assert.deepEqual(res.headers['content-location'],
-                'https://commons.wikimedia.org/api/rest_v1/page/html/File%3AThinkingMan_Rodin.jpg');
-        });
-    });
-
-    it('should not redirect if file is missing on commons', function() {
```

Comment:
They were duplicated in the redirects file. I've copied them but forgot to remove